src/ProjectTemplates/Web.ItemTemplates/content/MvcController/.template.config/template.json

```diff
@@ -46,7 +46,7 @@
     {
       "id": "openInEditor",
       "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
-      "description": "Opens the created controller in the editor",
+      "description": "Opens HomeController.cs in the editor",
```

Comment:
Same feedback regarding `HomeController.cs` as `ValueController` at https://github.com/dotnet/aspnetcore/pull/46886/files#r1119243746. When the user uses this template the generated file will be named by the user. `HomeController` is an implementation detail of the template and won't make sense to the end-user.